tools/decode-packets/src/main/scala/xtoolspar.scala

```diff
@@ -0,0 +1,274 @@
+/**
+  * Created by SouNourS on 20/12/2016.
+  */
+
+// Make sure the input files have UTF8 encoding!
+
+import java.io.{BufferedWriter, File, FileWriter}
+import java.nio.charset.CodingErrorAction
+
+import net.psforever.packet._
+import scodec.Attempt
+import scodec.bits._
+
+import scala.io.{Codec, Source}
+import scala.collection.parallel.CollectionConverters._
+
+object Xtoolspar {
+
+  def main(args: Array[String]): Unit = {
```

Comment:
Do we have to keep this around? If we wanted to preserve a copy we could make it part of the git history, but there's no reason someone would use this over the improved version, right? Because when you run `sbt decodePackets/run` right now you get a `Multiple main classes detected` warning, I don't think you're supposed to have multiple mains per project.

Reply:
Interesting, I was running it through IDEA so wasn't getting those errors. The thinking behind keeping the original (for the moment) would be so you can run both and compare the outputs ensure there's not any unexpected differences between the two.

Reply:
It lets you select which one you want to it's not a big deal, we can keep it for now.